### DISCUSSION TITLE
Long range at sea : 10-20 km expected, 300m obtained !!

### DISCUSSION THREAD

**Discussion by quentinorh** [Lora-net/LoRaMac-node#1528] [Q&A]
Hi,

I'm trying to communicate in Lora with Adafruit Feather M0 in 433mhz.

The objective is to be able to transmit the GPS position of a beacon in the open sea, and to find it by boat, probably with a Yagi antenna.

I have two M0 433mhz cards, one with a 16.5cm wire antenna, the other with a yagi antenna (9 dBi) and a 5w power amplifier (see photos).

![336591562_767149548330635_6955466369325810350_n](https://user-images.githubusercontent.com/30799378/227777798-68c4821d-b43b-4055-ade1-691d5fc1bfb4.jpg)
![61eClBQyujL _AC_SL1001_](https://user-images.githubusercontent.com/30799378/227777513-303dc6d8-5fc8-4e21-b891-11dcad5eec20.jpg)

I have just done some tests, I can't manage to receive a package at more than 300m... whereas I would need 10-20km!

Am I too optimistic, or am I wrong in the setting / electronics?

**Comment by danak6jq** [comment]
You're asking a radio question, not so much a LoRaWAN question.

I note that amplifier you're using (I believe I have the same one) is incomplete without (considerably) more low-pass filtering on the output. In fact, I'm not even sure what the performance of that amplifier at 433MHz is in terms of output matching. Have you verified that you're not over-driving it, and that you're switching it out of the antenna path on receive (this may explain the limited range right there)?

**Reply by quentinorh** [reply]
Yes, I removed the amplifier and it already works better!

**Comment by mluis1** [comment]
As mentioned by @danak6jq this question is not related to this project. This project is a software library implementing the LoRaWAN protocol stack.

If you do not mind I move this question to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

A better place to ask these kind of questions is the [LoRa Developers forum](https://forum.lora-developers.semtech.com/)

**Comment by quentinorh** [comment]
Now that it works better, would you have any software settings to share with me to optimize the range?

On the hardware side, do you have better antennas or solutions to amplify the signal received?

**Reply by danak6jq** [reply]
I'd suggest using a ready-made high-power 430MHz module; eByte has a wide selection of LoRa modules, including high-power 430MHz modules such as the 2W  https://www.cdebyte.com/products/E22-400M33S which combines an SX1268 with a 2W power amplifier. I've ported LoRaMac-node to the E22-900M30S and E19-915M30S with excellent results. I attached the E22-900M30 to an ST Nucleo STM32L152RE, and ported to the Atmel SAMD21 on the Sparkfun SparkX Pro RF 1W. These experimental ports are available as a fork of LoRaMac-node at https://github.com/meterme/LoRaMac-node/tree/local-devices (this is an experimental branch, the result of my own interests, no warranty, etc. - contact me with questions).

Note: it's _very important_ to measure and calibrate the power settings of the SX radio when using these amplified modules - using the "standard" settings will severely overdrive the power amplifier in the module and not provide accurate TXpower control for LoRaWAN as well as compliance with radio regulations (I'm assuming you are aware of and complying with relevant radio regulations where you are). I found the E19-915M30S has 17dB of PA gain and E22-900M30S has 8dB of PA gain for output power over +10dBm, and 10dB below that.

You'll need to do a similar power measurement for the 430MHz modules. I used the tx-cw example from the LoRaMac-node applications to drive steady power for measurement.

I'd suggest a proper 430MHz antenna for the remote node, a piece of wire hanging-off without a ground-plane is likely to be unpredictable. That's a nice beam you already have.
